index.html

```diff
@@ -9,6 +9,20 @@
   <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
   <link rel="manifest" href="/site.webmanifest">
   <title>Deal With It GIF emoji generator</title>
+  <meta property="og:type" content="website" />
+  <meta property="og:title" content="Deal With It GIF emoji generator" />
+  <meta property="og:description" content="All done artisanally and securely in your browser." />
+  <meta property="og:url" content="https://emoji.build/deal-with-it-generator/" />
+  <meta property="og:site_name" content="Deal With It GIF emoji generator" />
+  <meta property="og:image" content="https://emoji.build/deal-with-it-generator/favicon.ico" />
+  <meta property="og:image:width" content="200" />
+  <meta property="og:image:height" content="200" />
+  <meta property="og:image:secure_url" content="https://emoji.build/deal-with-it-generator/favicon.jpg" />
+  <meta name="twitter:card" content="summary" />
+  <meta name="twitter:domain" content="emoji.build" />
+  <meta name="twitter:title" content="Deal With It GIF emoji generator" />
+  <meta name="twitter:description" content="All done artisanally and securely in your browser." />
```

Comment:
```suggestion
  <meta name="twitter:description" content="Most over-engineered Deal With It emoji generator - all done securely in your browser for free." />
```

Something like this might be a better description.

Reply:
Sure thing! I wasn't sure to what degree I wanted to exercise editorial creativity over someone else's site, so wherever possible, I copy/pasted existing assets. Happy to use this instead!